### DISCUSSION TITLE
My code editor (atom) doesn't recognize css tags.  HELP :(

### DISCUSSION THREAD

**Discussion by IaraNeumann** [community/community#28245] [Codespaces]
I'm new to code and I don't understand why when writing css the autocomplete doesn't appear.
I have all the packages installed (autocomplete-css and autocomplete-plus...etc) 
All the code comes out in gray, please help 😭

Screenshot👇🏻

(https://im.ge/i/F3Q9NW)

**Comment by theglobehead** [comment]
Hei! 
Perhaps, it is due to the body tag not being closed.
Your example:
```
body{
    background-color: orange;

  hr {
      backgroung-color: blue;
  }
```

Correct: 
```
body{
    background-color: orange;
}

hr {
    backgroung-color: blue;
}
```

If this does not work, you could try a different editor (for instance VS code) 

Or you could try adding separate styles.css file and linking it like this:
```
<head>
  <link rel="stylesheet" href="styles.css">
</head>
```

**Reply by airtower-luna** [reply]
Trying a different editor might be good either way, considering Atom has been deprecated: https://github.blog/2022-06-08-sunsetting-atom/
